docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -85,6 +85,20 @@ You can edit your canvas inline while in the context of dashboard mode.
 
 {{< video-embed src="/static/img/docs/canvas-panel/canvas-inline-editor-9-2-0.mp4" max-width="750px" caption="Inline editor demo" >}}
 
+### Pan and zoom
+
+You can enable panning and zooming in canvas. This allows you to both create and navigate more complex designs.
+
+{{% admonition type="note" %}}
+To try out pan and zoom, enable the `canvasPanelPanZoom` [feature toggle](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/feature-toggles/#preview-feature-toggles).
+{{% /admonition %}}
```

Comment:
What release stage is this in? My understanding is that if it's in in private preview, it really shouldn't be in the What's new (even though we offer that option in the availability selections). For the docs, we have shortcodes with standardized text. Just use one of these and replace the text in the quotation marks. As you can see, the one for private preview doesn't include the option to enable a feature a flag because that information wouldn't generally be public.

```
{{< docs/experimental product="experimental-feature" featureFlag="its-feature-flag" >}}
{{< docs/private-preview product="private-preview-feature" >}}
{{< docs/public-preview product="public-preview-feature" featureFlag="its-feature-flag" >}}
```

You can see the text that each of these shortcodes generates here:
https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsexperimental
https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsprivate-preview
https://grafana.com/docs/writers-toolkit/write/shortcodes/#docspublic-preview

Reply:
I believe we settled on public preview (see [release life cycle docs for reference](https://grafana.com/docs/release-life-cycle/))

Reply:
Okay I think there was an internal note in the What's new that made me unsure. So I'd go with this:
```suggestion
{{< docs/public-preview product="Canvas pan and zoom" featureFlag="`canvasPanelPanZoom`" >}}
```

It'll display in the docs as:
**Note**: Canvas pan and zoom is currently in [public preview](https://grafana.com/docs/release-life-cycle/). Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available. Enable the `canvasPanelPanZoom` feature toggle in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.